pricing-page-exercise/tugba/styles.css

```diff
@@ -0,0 +1,215 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-family: "Open Sans", sans-serif;
+    font-size: 16px;
+}
+body {
+    margin: 0;
+    padding: 0;
+  background-color: grey;
+}
+/* navbar section */
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.25rem;
+    background: rgb(180, 180, 180);
+    height: 70px;
+    border-bottom: 1px solid rgb(222 226 230);
+    box-shadow: rgba(0, 0, 0, 0.05) 0px 7px 11px 0px;  
+}
+
+.heading-small {
+    font-size: 1.2rem;
+    font-weight: 400;
+    text-align: left;
+}
+
+.navbar ul {
+    display: flex;
+    align-items: center;
+    gap: 1.2rem; 
+    list-style-type: none;
+}
+
+.navlink {
+    text-decoration: none;
+    color: #363636;
+    font-size: 1rem;
+}
+.navlink:hover {
+    text-decoration:underline;
+    color:#363636;
+}
+
+.text-center {
+    text-align: center;
+  font-weight: bold;
+}
+
+
+.btn {
+    color:rgb(30, 143, 218);
+    border: 1px solid rgb(30, 143, 218);
+    border-radius: 5px;
+    padding: 0.6rem; 
+    text-decoration: none;   
+}
+.btn:hover {
+    background-color: rgb(30, 143, 218);
+    color:white;
+}
+.btn:active {
+    box-shadow: 0 0 2px 2px #3e8fc5;
+}
+
+
+/* description section */
+.description {
+    background: (255, 255, 255);
+    max-width: 700px;
+    width: 70%;
+    padding: 0.5rem;
+    margin-bottom: 4rem;
+    margin-top: 4rem;
+    margin-left: auto;
+    margin-right: auto;
+}
+.description h1 {
+    margin-bottom: 0.6rem;
+    font-size: 3.3rem;
+    font-weight: 200;
+    opacity: 0.9;
+}
+.description p {
+    letter-spacing: 0.7px;
+    line-height: 1.7rem;
+    opacity: 0.9;
+    color:#363636;
+    font-size: 1.1rem;
+
+}
+/* content section */
+.total-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+}
+.content {  
+    margin-bottom: 4rem;
+    display: flex;
+    flex-direction: row;
+    gap: 1.8rem;
+    justify-content: center;
+    align-items: flex-start;
+    flex-wrap: wrap;
+}
+.prices {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    min-width: 18rem;
+
+}
+.card_head {
+    width: 100%;
+    height: 3rem;
```

Comment:
You can of course specify height, width, min-width (as above) using rem but it may not be as easy to understand for the one reading your code. I'd stick to px, % or vh/vw for width and height instead. 

Also, you probably don't want to set height like this. It's usually better to add enough margin/padding and let the content of the div decided how much height it should take.